src/documentation/basic-opts.page

```diff
@@ -12,17 +12,17 @@ This document covers the basic operations provided by Uniqush. It won't cover th
 
 ## How to Communicate with Uniqush
 
-Uniqush runs as a stand along program, which means you don't need to write any code to link against uniqush. Uniqush listens a port when it is running. You can change the port in its configuration file. To send request to uniqush, you simply need to connect to that port, send an HTTP Post request with parameters inside the body.
+Uniqush runs as a stand alone service, which means you don't need to write any code to link against uniqush. Uniqush listens on a TCP port when it is running. You can change the port in its configuration file. To send request to uniqush, you simply need to connect to that port, send an HTTP POST request with the parameters inside the body.
 
 ## Adding Push Service Providers to a Service
 
-Without telling uniqush infomation about push service provider, a server can never send data through uniqush. Usually, to use a push service provider, uniqush need to know the type of that push service provider (e.g. [GCM], [APNS], Black Berry Push Service or Microsoft Push Notification Service for Windows Phone) and authentication information -- for example, authentication token for [GCM]; private/public keys for [APNS], etc.
+Without telling uniqush infomation about a push service provider, a server can never send data through uniqush. Usually, to use a push service provider, uniqush need to know the type of that push service provider (e.g. [GCM], [APNS], Black Berry Push Service or Microsoft Push Notification Service for Windows Phone) and authentication information -- for example, authentication token for [GCM]; private/public keys for [APNS], etc.
```

Comment:
Aside: this does not clearly indicate that Microsoft and Blackberry aren't implemented.

It also doesn't mention ADM.